website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-supported-dbt-versions.md

```diff
@@ -1,13 +1,29 @@
 ---
-title: "Supported dbt versions"
```

Comment:
Can you:
- [ ] update the file name to match — the `id` keys were transported over when we moved things across from readme.io, but default to the filename value, so we really didn't need them. Just for the sake of simplicity, it would be good to keep them the same
- [ ] Add an entry to `_redirects` for the changed file

Reply:
Ah, yes. Good catch!